server/websocket.go

```diff
@@ -1378,7 +1377,8 @@ func (c *client) wsCollapsePtoNB() (net.Buffers, int64) {
 			for i := 0; i < len(nb); i++ {
 				b := nb[i]
 				if total+len(b) <= mfs {
-					bufs = append(bufs, b)
+					buf := nbPoolGet(len(b))
+					bufs = append(bufs, append(buf, b...))
 					total += len(b)
 					continue
```

Comment:
Should we not have a `nbPoolPut(nb[i])` (or `nbPoolPut(b)`) here since we are using `continue`?

Reply:
Good spot, added.